src/ripple/protocol/impl/ErrorCodes.cpp

```diff
@@ -174,7 +174,7 @@ Json::Value make_error (error_code_i code, std::string const& message)
 
 bool contains_error (Json::Value const& json)
 {
-    if (json.isObjectorNull() && json.isMember (jss::error))
+    if (json.isObject() && json.isMember (jss::error))
```

Comment:
looks like `isMember` already does `isObject`.

Reply:
Yeah, I noticed that part way through this exercise.  We do this in a ton of places.  I thought about fixing all of them in this P/R but was worried that would be a bridge too far.  However I could go there in this P/R just under a separate commit if the team is behind that simplification.